src/routeGeneration/templates/koa.ts

```diff
@@ -146,9 +142,7 @@ export function RegisterRoutes(router: any) {
             next();
         })
         .catch((error: any) => {
-            context.status = error.status || 500;
-            context.body = error;
-            context.throw(context.status, error.message, error);
+            context.throw(error.status, JSON.stringify({message: error.message}));
```

Comment:
I think in some cases (like validation errors), a lot of information about the issue is contained within the error object itself.  I wonder if it makes sense to continue sending the entire error object, as some people may have been exploiting the fact that the error would be within the body of the response.